crates/levm/src/call_frame.rs

```diff
@@ -2,6 +2,13 @@ use crate::{memory::Memory, opcodes::Opcode};
 use bytes::Bytes;
 use ethereum_types::{Address, U256};
 
+#[derive(Clone, Debug, Default, Eq, PartialEq, Hash)]
+pub struct Log {
+    pub address: Address,
+    pub topics: Vec<U256>,
```

Comment:
```suggestion
    pub topics: Vec<H32>,
```

I think the "correct" type to use here is H32: https://github.com/ethereum/execution-specs/blob/51fac24740e662844446439ceeb96a460aae0ba0/src/ethereum/cancun/blocks.py#L91

Reply:
Oh you are right, just used the `U256` like the default, changing for that 🫡